week-3/todo-list/index.html

```diff
@@ -3,25 +3,29 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>To_do-list</title>
     <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+    />
   </head>
   <body>
-    <form>
-      <div>
-        <input type="text" placeholder="New todo..." />
+    <div class="wrapper">
+      <header>Todo-List</header>
+      <div class="inputField">
+        <input type="text" placeholder="Add your new Todo" />
```

Comment:
user's should enter the todo, and also set the deadline date, the date input field is missing